keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/EnvironmentSummary.kt

```diff
@@ -10,6 +11,7 @@ import com.netflix.spinnaker.keel.api.id
 /**
  * Summarized data about a specific environment, mostly for use by the UI.
  */
+@JsonPropertyOrder(value = ["name"])
```

Comment:
what is the purpose of only having one value here?

Reply:
It makes sure name is at the top. All other properties are arranged in whatever way Jackson thinks is the right way.

Reply:
oh cool!